<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatbox</class>
 <widget class="QDialog" name="chatbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 243, 243);</string>
  </property>
  <widget class="QLineEdit" name="nicknameC">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
border-radius:15px;
color: rgb(27, 75, 101);
</string>
   </property>
   <property name="placeholderText">
    <string comment="Rechercher" extracomment="Rechercher"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageC">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
border-radius:15px;
color: rgb(27, 75, 101);
</string>
   </property>
   <property name="placeholderText">
    <string comment="Rechercher" extracomment="Rechercher"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 243, 243);
color: rgb(27, 75, 101);
</string>
   </property>
   <property name="text">
    <string>CHAT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(27, 75, 101);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Surnom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(27, 75, 101);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendC">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
	
	background-color: rgb(255, 255, 255);
	
	border-image: url(:/2A3/qt/tsawer/envoyer-un-mail.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditC">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>491</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
border-radius:15px;
color :rgb(27, 75, 101);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bindC">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
border-image: url(:/2A3/qt/tsawer/connexion (1).png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
